docs/distributions/debian/installation.md

```diff
@@ -1,93 +1,97 @@
 # Introduction
 
-This guide shall help you in installing a Debian or Ubuntu based Linux distro, which is not available as a modified ISO with T2 support.
+This guide will help you in installing a Debian or Ubuntu-based Linux distro that does not have a modified ISO with T2 support.
 
-Some popular distros include :-
+Some popular distros include:
 
-1. [Linux Mint](https://linuxmint.com/)
-2. [Pop!_OS](https://pop.system76.com/)
-3. [elementary OS](https://elementary.io/)
-4. [Debian](https://www.debian.org/)
-5. [Zorin OS](https://zorin.com/)
-6. [Kali Linux](https://www.kali.org/)
-7. Other Ubuntu flavours like [Kubuntu](https://kubuntu.org/), [Ubuntu unity](https://ubuntuunity.org/) etc.
+* [Linux Mint](https://linuxmint.com/) (Ubuntu-based | Debian-based if using LMDE (Linux Mint Debian Edition))
+* [Pop!_OS](https://pop.system76.com/) (Ubuntu-based)
+* [elementary OS](https://elementary.io/) (Ubuntu-based)
+* [Debian](https://www.debian.org/) (Debian)
+* [Zorin OS](https://zorin.com/) (Ubuntu-based)
+* [Kali Linux](https://www.kali.org/) (Debian-based)
+* Other Ubuntu flavours like [Kubuntu](https://kubuntu.org/), [Ubuntu Unity](https://ubuntuunity.org/) etc. (Ubuntu-based)
 
-# Hardware Requirements
+# Required Hardware
 
-* USB-C to USB adapter
-* Wired internet connection (Ethernet/USB tethering) or Wi-Fi adapter compatible with Linux
+* USB-C to USB-A adapter
+* Internet connection - either **wired** (using an USB Ethernet dongle or USB tethering from a phone) or **wireless** (using a Linux-compatible Wi-Fi adapter)
 * External keyboard and mouse
 
 # Install Procedure
 
-!!! Warning "Installation fails when installing the bootloader (Eg :- GRUB)"
-    It is possible that during installation, when the installer is installing the bootloader (GRUB in most cases), the installation may fail. This is because the distro's ISO is using an old Linux kernel which doesn't support writing to the NVRAM of T2 Macs. In such a case, boot into the ISO again. When the initial menu gets displayed having options to try/install the distro, press "e" on the option you otherwise would have chosen to install. This will open the command line. Add `efi=noruntime` to the command line and press "F10" to boot. This should fix the issue.
+!!! Warning "Installation fails when installing the bootloader (e.g. GRUB)"
+    It is possible that during installation, when the installer is installing the bootloader (GRUB in most cases), that the installation may fail. This is because the distro's ISO is using an old Linux kernel which doesn't support writing to the NVRAM of T2 Macs. In such a case, boot into the ISO again. When the initial menu gets displayed having options to try/install the distro, press "e" on the option you otherwise would have chosen to install. This will open the command line. Add `efi=noruntime` to the command line and press "F10" to boot. This should fix the issue.
```

Comment:
I don't think there should be a that added here. Just read the sentence without brackets and you'll understand.